src/display/annotation_layer.js

```diff
@@ -445,40 +465,28 @@ class AnnotationElement {
    * are of a type that works with popups (such as Highlight annotations).
    *
    * @private
-   * @param {HTMLDivElement|HTMLImageElement|null} trigger
-   * @param {Object} data
    * @memberof AnnotationElement
    */
-  _createPopup(trigger, data) {
-    let container = this.container;
-    if (this.quadrilaterals) {
-      trigger ||= this.quadrilaterals;
-      container = this.quadrilaterals[0];
-    }
-
-    // If no trigger element is specified, create it.
-    if (!trigger) {
-      trigger = document.createElement("div");
-      trigger.classList.add("popupTriggerArea");
-      container.append(trigger);
-    }
-
-    const popupElement = new PopupElement({
-      container,
-      trigger,
-      color: data.color,
-      titleObj: data.titleObj,
-      modificationDate: data.modificationDate,
-      contentsObj: data.contentsObj,
-      richText: data.richText,
-      hideWrapper: true,
+  _createPopup() {
+    const { container, data } = this;
+    container.setAttribute("aria-haspopup", true);
+
+    this.popup = new PopupAnnotationElement({
```

Comment:
I'm probably missing something, but it's not clear to me where in the code we're accessing the `popup`-property on a regular `AnnotationElement`-instance?